provider/storage_benchmark.py

```diff
@@ -227,6 +227,17 @@ def clean(self, timeout=1800, force=False):
         if self.env_files:
             self._remove_env_files()
 
+    @classmethod
+    def _clean_env(cls, func):
+        """ Decorator that clean the env files. """
+        def __clean_env(self, *args):
```

Comment:
better to use `functools.wraps` to decorate `__clean_env`, also, it is better defined as `__clean_env(self, *args, **kargs)`

Reply:
Thanks, have updated.